initial-ai-src/index.html

```diff
@@ -0,0 +1,104 @@
+          />
+
+          <label for="zone-selection">Which zones do you travel in?</label>
+          <select id="zone-selection">
+            <option value="1">Zone 1 (Oslo city center)</option>
+            <option value="2" selected>Zones 1-2 (Oslo + inner suburbs)</option>
+            <option value="3">Zones 1-3 (Oslo + outer suburbs)</option>
+            <option value="4">Zones 1-4 (Oslo + extended area)</option>
+            <option value="all">All zones (Oslo + Akershus)</option>
+          </select>
+
+          <label for="current-discount"
+            >What's your current Reis discount level?</label
+          >
+          <select id="current-discount">
+            <option value="0" selected>
+              0% - I'm starting fresh (first 4 tickets)
+            </option>
+            <option value="5">5% - I've bought 5-9 tickets recently</option>
+            <option value="10">10% - I've bought 10-14 tickets recently</option>
+            <option value="15">15% - I've bought 15-19 tickets recently</option>
+            <option value="20">20% - I've bought 20-24 tickets recently</option>
+            <option value="25">25% - I've bought 25-29 tickets recently</option>
+            <option value="30">30% - I've bought 30-34 tickets recently</option>
+            <option value="35">35% - I've bought 35-39 tickets recently</option>
+            <option value="40">
+              40% - I have maximum discount (40+ tickets recently)
+            </option>
+          </select>
+
+          <label for="calculation-period">Calculate for:</label>
+          <select id="calculation-period">
+            <option value="1">24 hours (1 day)</option>
+            <option value="7">7 days (1 week)</option>
+            <option value="30" selected>30 days (1 month)</option>
+            <option value="365">365 days (1 year)</option>
+          </select>
+
+          <button id="calculate-btn">Calculate Discount</button>
+        </div>
+
+        <div class="results-section" id="results">
+          <!-- Results will be populated by JavaScript -->
+        </div>
+
+        <div class="comparison-section" id="comparison">
+          <!-- Period ticket comparison will be populated by JavaScript -->
+        </div>
+
+        <div class="break-even-section" id="break-even">
+          <!-- Break-even analysis will be populated by JavaScript -->
+        </div>
+
+        <div class="info-section">
+          <h3>How Reis Discounts Work:</h3>
+          <ul>
+            <li>First 4 tickets: <strong>0% discount</strong></li>
+            <li>Tickets 5-9: <strong>5% discount</strong></li>
+            <li>Tickets 10-14: <strong>10% discount</strong></li>
+            <li>Tickets 15-19: <strong>15% discount</strong></li>
+            <li>And so on... up to <strong>40% maximum discount</strong></li>
+          </ul>
+          <p>
+            <small
+              >* Discounts are calculated based on tickets purchased in the last
```

Comment:
🐛 This feels like a pretty important side-note. Should it really be just a `<small>` in the fine print?